src/renderer/screens/exchange/Swap2/Form/FormLabel.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import React from "react";
+import Text from "~/renderer/components/Text";
+
+export function FormLabel({ children }: { children?: React$Node }) {
+  return (
+    <Text uppercase fontWeight="600">
```

Comment:
use `import Text from "~/renderer/components/Text";`
and something like (don't quote me on the values)

`<Text ff="Inter|SemiBold" fontSize={18} color="palette.text.shade100">`

Don't know the specifics. Also, make sure it works with the different themes
You can start the app with `DEBUG_THEME=1` env and see if it breaks the UI or not